examples/datasets/dataloader.py

```diff
@@ -0,0 +1,46 @@
+from math import atan2, cos, radians, sin, sqrt
+import pandas as pd
+import networkx as nx
+from mis import MISInstance
+
+class DataLoader:
+    def distance_from_coordinates(self, a_1 : tuple[float, float],a_2 : tuple[float, float]) -> float:
+        # Radius of the Earth in kilometers
+        R = 6371.0
+        # Convert latitude and longitude from degrees to radians
+        lat1, lon1 = radians(a_1[0]), radians(a_1[1])
+        lat2, lon2 = radians(a_2[0]), radians(a_2[1])
+
+        # Differences in coordinates
+        dlat = lat2 - lat1
+        dlon = lon2 - lon1
+
+        # Haversine formula
+        a = sin(dlat / 2)**2 + cos(lat1) * cos(lat2) * sin(dlon / 2)**2
+        c = 2 * atan2(sqrt(a), sqrt(1 - a))
+
+        # Distance in kilometers
+        distance_km = R * c
+        return distance_km
+
+    def load_from_csv_coordinates(self, file_path : str):
```

Comment:
The return type is missing. Also, could you please use `Path` from `pathlib` ?

Reply:
Please also document and add docstrings.

Reply:
Done